### PR TITLE
Disable tests with race failure

### LINKED ISSUE #3206: Failures in race tests
Failed tests in this run have to be disabled until fixed properly:
https://circleci.com/gh/istio/istio/23976

Your build ran 3174 tests with 10 failures
TestBatchMeasurements - appoptics
Failed
TestBatchMeasurements/All_Good - appoptics
Failed
TestLog - papertrail
Failed
Less
TestLog/All_Good - papertrail
Failed
TestNewMetricsHandler - solarwinds
Failed
TestNewMetricsHandler/All_good - solarwinds
Failed
TestPersistBatches - appoptics
Failed
TestPersistBatches/Persist_all_good - appoptics
Failed
TestStressEnvoy - stress
Failed
TestTCPMixerFilter - tcp_filter

### PR DESCRIPTION
**Pull request by andraxylia**
Fixes #3206

### CODE REVIEW

**Comment by andraxylia** [comment]
pilot-e2e failures are unrelated, caused by grpc tests #3222

**Comment by andraxylia** [comment]
e2e-smoke failures caused by #3221

File: mixer/test/client/stress/stress_test.go [hunk 1]
```diff
@@ -21,6 +21,8 @@ import (
 	"testing"
 	"time"
 
+	"os"
```
Comment (douglas-reid) [inline mixer/test/client/stress/stress_test.go]: this is a fmt issue. i don't know why the linters aren't checking this, but please run `bin/fmt.sh` all files in this PR.

Comment (andraxylia) [reply mixer/test/client/stress/stress_test.go]: did by mistake on all files and now I got a bunch of changes :(

**Comment by andraxylia** [comment]
done, added issue # and run fmt, please approve so I can get this in.

I see fmt needs to be run on all files, I will submit a PR just with fmt.

**Pull request merged**